### DISCUSSION TITLE
post/2024/mlnscript-r2mlwin-apple-silicon/

### DISCUSSION THREAD

**Discussion by giscus** [remlapmot/remlapmot.github.io#1] [Announcements]
# post/2024/mlnscript-r2mlwin-apple-silicon/

How to run MLwiN natively on Apple Silicon Macs via R and the R2MLwiN package.

https://remlapmot.github.io/post/2024/mlnscript-r2mlwin-apple-silicon/

**Comment by kwongsiufung** [comment]
Worked like a charm, although didn't need to do anything in the terminal. Thanks Tom!

**Comment by remlapmot** [comment]
That's great @kwongsiufung
